std-bits/aws/src/main/java/org/enso/aws/ClientBuilder.java

```diff
@@ -59,6 +61,18 @@ public S3Client buildS3Client() {
         .build();
   }
 
+
+  /**
+   * Signs an AWS request using the provided credentials.
```

Comment:
```suggestion
   * Creates a HttpClient that signs requests using the provided credentials.
```

Reply:
This method does not do the signing itself, IMO the current comment was a bit misleading.

Reply:
As noted doc comments not done.

Reply:
Ah ok, sorry I missed that.

Reply:
Related, what do you plan to do about?
> AccessKey is returned in header plain text, will reject if credential is AccessKey/SecretKey and AccessKey is secret.

Looking through the code - I did not find any special handling for this.
Shall we just remove the access key from the response if it was provided as a secret? We could replace the response header with `Access Key: __SECRET__` instead of actual value.

Reply:
The access key is inside the authorization header so shouldn't be in the response.
The original prototype had it returning into a list of extra headers. 
I think with the HTTPClient approach it shouldn't leak out.